### PR TITLE
refactor: AccountModal

### PR DESCRIPTION
**Pull request by anton-karlovskiy**

### CODE REVIEW

File: src/common/components/account-modal/account-modal.tsx [hunk 1]
```diff
@@ -31,81 +37,77 @@ function AccountModal(props: AccountModalProps): ReactElement {
...
-                      height='30'
-                      alt='wallet-logo' />
-                    <span className='name'>Polkadot.js</span>
-                  </div>
+                  &nbsp;{t('here')}
                 </a>
-              </div>
-            </div>
+                .
+                {t('refresh_page')}
```
Comment (nud3l) [inline src/common/components/account-modal/account-modal.tsx:71]: I think this refresh page ended there by accident (was already there before your PR). Can you please delete it?

image

Comment (anton-karlovskiy) [reply src/common/components/account-modal/account-modal.tsx:71]: Done!

Comment (nud3l) [reply src/common/components/account-modal/account-modal.tsx:71]: Did you push? The `  {t('refresh_page')}` is still there ;)

Comment (anton-karlovskiy) [reply src/common/components/account-modal/account-modal.tsx]: Sorry, I was mistaken.
I pushed this time. :)

**Review by nud3l — APPROVED** [review]

**Pull request merged**